docs/segment.md

```diff
@@ -2,98 +2,188 @@
+## Text regions segmentation
 
-The segmentation panel is toggled by clicking on the "Segmentation" button at the top of the page (or by pressing ++control+3++).
+In the following example, we segmented and labeled the zones that bear a semantic role for our comprehension of the text. We are dealing with a poem, so we drew with eScriptorium bounding boxes around the page number (in blue), the title (in purple), and the stanzas (in green).
+
+However, you could have decided to annotate the layout more precisely, for example, all the verses inside the stanzas. It is up to you to decide how deep you want to go into the text regions segmentation, depending on your needs.
+
+![image: Illustration of several segmented text regions](img/segment/text_regions_segmentation.png "Several segmented text regions are visualized with eScriptorium interface")
+
+### Segmenting text regions with eScriptorium 
+
+You can draw manually draw bounding boxes inside the Segmentation panel. To do so, switch from baseline editing mode to region mode, by clicking the **mosaic** button, or by pressing ++r++.
+
+You can now draw bounding boxes around the text regions you want to annotate by clicking and dragging a zone (green by default).
+
+![image: Drawing bounding boxes in the Segmenting panel of eScriptorium](img/segment/bounding_boxes_drawing.gif "Several bounding boxes are drawn around meaningful text regions.")
+
+You can modify a text region by clicking on it, and by dragging it to the dimensions you want.
+
+![image: Resizing a text region](img/segment/resizing_text_region.gif "")
+
+You can delete a text region by clicking on it, still in region mode, and by clicking on the delete button, or by pressing ++delete++.
+
+!!! note
+    Keep in mind that you can always press ++ctrl+z++ to undo any modification.
+
+### Labeling segmented text regions with an ontology
+
+eScriptorium allows to label segmented text regions with an ontology that you can customize.
+
+The ontology panel is one of the tabs you can access once you are working in a document. It is accessible with (`{base_url}/document/{document-id}/ontology/`). Text regions have a dedicated field named *region types*.
+
+![image: Ontology tab](img/segment/ontology_tab.png "Illustration of the ontology tab.")
+
+eScriptorium uses four region types by default:
+
+- Commentary
+
+- Illustration
+
+- Main
+
+- Title
+
+You can use them, or create your own. To deactivate default region types, you can deactivate them by unticking the ones you want to discard, and then by clicking on the update button.
+
+![image: Deactivating default text regions](img/segment/default_text_regions.gif "Modifying the text region ontology by deactivating default region types.")
+
+You can then create your own by writing the text region name you want into the dedicated field and then the + button. Don't forget to click on the update button to save your modifications!
```

Comment:
"and then click on the + button" --> otherwise, you are still with the verb "writing", which does not make sense